runtime/index.ts

```diff
@@ -1,2 +1 @@
 export * from './bootstrapper';
-export * from './config';
```

Comment:
Consumers should no longer import `SkyAppConfig` from `@skyux-sdk/builder/runtime` (instead, `@skyux/config`). I did a quick search and none of our GitHub libraries are referencing `runtime/config` so I'd guess it's not being referenced much at all (this was mostly left in as a backwards compatibility feature of Builder 1.x).